services/workflows-service/src/workflow/types/index.ts

```diff
@@ -11,3 +12,8 @@ export type CompleteWorkflowData = WorkflowRuntimeData & {
 };
 
 export type TEntityType = 'endUser' | 'business';
+
+export type WorkflowWithRelations = WorkflowRuntimeData & {
```

Comment:
Make sure to prepend types with `T` unless it's a type we expect consumers to import, for example from our packages or SDKs.